arcgis-ios-sdk-samples/Maps/Display layer view state/README.md

```diff
@@ -1,23 +1,25 @@
 # Display layer view state
 
-Determine if a layer is currently being viewed.
+Determine whether a layer is currently visible.
 
 ![Image of display layer view state](display-layer-view-state.png)
 
 ## Use case
 
-The view state includes information on the loading state of layers and whether layers are visible at a given scale. For example, you might change how a layer is displayed in a layer list to communicate whether it is being viewed in the map: Show a loading spinner next to its name when the view state is `loading`, gray out the name when `notVisible` or `outOfScale`, show the name normally when `active`, or with an error icon when the state is `error`.
+The view status includes information on the loading state of layers and whether layers are visible at a given scale. You might change how a layer is displayed in a layer list to communicate whether it is being viewed in the map. For example, you could show a loading spinner next to its name when the view status is `loading`, gray out the name when `notVisible` or `outOfScale`, show the name normally when `active`, or with an error icon when the state is `warning` or `error`.
 
 ## How to use the sample
 
-Pan and zoom around in the map. Each layer's view status is displayed. Notice that some layers configured with a min and max scale change to `outOfScale` at certain scales.
+When the feature layer is loaded, pan and zoom around the map. Note how the `AGSLayerViewState` flags change; for example, `outOfScale` becomes true when the map is scaled outside of the layer's min and max scale range. Tap the toggle to hide the layer and observe the view state change to `notVisible`.
+
+Disconnect from the network and pan around the map to see the `warning` status of the different layers when they cannot online fetch data. Reconnect to the network to see the warning disappear.
```

Comment:
"they cannot online fetch data" doesn't sound right. Should it be "they cannot fetch online data"?

Reply:
😂 It was a no-brainer copy-paste... You are right, I propose the same fix in `/common-samples/pull/2139#discussion_r490700`